--- a/squashfs-tools/mksquashfs.c       2021-02-08 10:08:42.135709202 +0800
+++ b/squashfs-tools/mksquashfs.c       2021-02-08 10:09:03.263649419 +0800
@@ -52,6 +52,7 @@
 #include <regex.h>
 #include <fnmatch.h>
 #include <sys/wait.h>
+#include <sys/sysmacros.h>
 
 #ifndef linux
 #ifndef __CYGWIN__
--- a/squashfs-tools/unsquashfs.c       2021-02-08 10:08:48.335691509 +0800
+++ b/squashfs-tools/unsquashfs.c       2021-02-08 10:09:24.919589603 +0800
@@ -30,6 +30,7 @@
 #include "xattr.h"
 
 #include <sys/types.h>
+#include <sys/sysmacros.h>
 
 struct cache *fragment_cache, *data_cache;
 struct queue *to_reader, *to_deflate, *to_writer, *from_writer;
